oak-doc/src/site/site.xml

```diff
@@ -152,10 +152,12 @@ under the License.
       <item href="FAQ.html" name="FAQ" />
     </menu>
     <menu name="Developing Oak">
-      <item href="dev_getting_started.html" name="Getting Started" />
+      <item href="development/dev_getting_started.html" name="Getting started" collapse="false">
+        <item href="development/developing-with-git.html" name="Developing with Git" />
+        <item href="development/nodestore-fixtures.html" name="Nodestore Fixtures" />
+      </item>
```

Comment:
I think the disadvantage of rearranging existing pages is that will result in broken links - if these pages were linked to at all. I guess that wouldn't be that much of a problem - but maybe we could find an alternative indeed. Eg we could also add `nodestore-fixtures.html` at the same level as the rest. (I tend to agree that it would probably be nicer structure otherwise, but then again it breaks existing links...)

Reply:
Ok, I moved the page on the same level as Getting started and others. I renamed the page to 'Testing' so it's more generic.